src/functions/uploadToRSIS/framework/repo/database.ts

```diff
@@ -33,17 +38,23 @@ export const createConnection = async (config: Config): Promise<Connection | und
  * @param bindValues   The bind variables values
  * @throws Various errors that must be caught and logged, including if expected rows proved to be different
  */
-export const execute = async (connection: Connection, sqlQuery: string, expectedRows: number, bindValues?: any):
+export const execute =
+  async (connection: Connection, sqlQuery: string, expectedRows: number, appRef: number, bindValues?: any):
     Promise<void> => {
-  debug(`Executing statement: \n***\n${sqlQuery}\n***`);
+    debug(`Executing statement: \n***\n${sqlQuery}\n***`);
+    let result;
+    try {
+      result = await connection.execute(sqlQuery, bindValues || {}, { autoCommit: true });
 
-  const result = await connection.execute(sqlQuery, bindValues || {}, { autoCommit: true });
+      debug(`${result.rowsAffected} rows updated`);
 
-  debug(`${result.rowsAffected} rows updated`);
+      if (result.rowsAffected !== expectedRows) {
+        // tslint:disable-next-line:max-line-length
```

Comment:
Enable linting and use line break

Reply:
Line breaks mess up the error logs and add new line to it

Reply:
It adds a `\n` to the error message...

Reply:
If I use line break